pages/myinfo/bookmark/api/useInfiniteMyInfoBookmark.ts

```diff
@@ -38,15 +42,19 @@ export const useInfiniteMyInfoBookmark = (sortOption: MyinfoBookmarkDropdownProp
   } = useInfiniteQuery({
     queryKey: ['techBlogBookmark', sortOption],
     // 데이터를 요청하는데 사용하는 함수
-    queryFn: ({ pageParam }) =>
-      getTechBlogData({
-        techArticleId: pageParam,
-        bookmarkSort: sortOption,
-      }),
-    initialPageParam: '',
+    queryFn: ({ pageParam }) => {
+      if (!isValidSortOption) {
+        return Promise.resolve({ data: { content: [], last: true } });
+      } else {
+        return getTechBlogData({
+          techArticleId: pageParam,
+          bookmarkSort: sortOption,
+        });
+      }
+    },
```

Comment:
이 부분 early return으로 수정하는것 어떤가용?? 더 눈에 잘 들어오는 거 같아서용!

```suggestion
    queryFn: ({ pageParam }) => {
      if (!isValidSortOption) {
        return Promise.resolve({ data: { content: [], last: true } });
      }

      return getTechBlogData({
        techArticleId: pageParam,
        bookmarkSort: sortOption,
      });
    },
```

Reply:
헐 `Suggested change`라는 기능이 있다니 짱이당

Reply:
[65c125d](https://github.com/dreamyPatisiel/devdevdev-client/pull/86/commits/65c125d262854b39c0063da628ae67f3e777d59f)  😊3개 한꺼번에 커밋해놧어용